src/plugins/walletConnectToDapps/utils/createApprovalNamespaces.ts

```diff
@@ -0,0 +1,80 @@
+import type { ChainId } from '@shapeshiftoss/caip'
+import { fromAccountId } from '@shapeshiftoss/caip'
+import { isEvmChainId } from '@shapeshiftoss/chain-adapters'
+import type { ProposalTypes, SessionTypes } from '@walletconnect/types'
+import { uniq } from 'lodash'
+
+import { EIP155_SigningMethod } from '@/plugins/walletConnectToDapps/types'
+
+export const createApprovalNamespaces = (
```

Comment:
Mostly a copypasta and cleanup job from the current `_createApprovalNamespaces` (now living here)